core/controllers/admin.py

```diff
@@ -60,6 +62,9 @@
 class AdminPage(base.BaseHandler):
     """Admin page shown in the App Engine admin console."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
```

Comment:
Is there any issue with having `{}` to be the default value of `URL_PATH_ARGS_SCHEMAS`?